Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/extractors/BaseTimeExtractor.java

```diff
@@ -88,6 +88,13 @@ public BaseTimeExtractor(ITimeExtractorConfiguration config) {
 
             Match[] matches = RegExpUtility.getMatches(regex, text);
             for (Match match : matches) {
+                
+                // @TODO Remove when lookbehinds are handled correctly
```

Comment:
Can you make this a method?

Reply:
mmm, but the if statement returns a "continue" when matched and as far as I know it is not possible to return an instruction from a method in Java. So, I could put the current if condition in a method which returns a boolean and then add a new if statement that "continue" when the method returns "true". Is that what you mean?

Reply:
Yes, as I think it would make the code a little more readable and it then could be reused in other filtering or in lambdas.

Reply:
done.